lnwallet/interface_test.go

```diff
@@ -476,52 +457,6 @@ func testDualFundingReservationWorkflow(miner *rpctest.Harness, wallet *lnwallet
 	if !bytes.Equal(channels[0].FundingOutpoint.Hash[:], fundingSha[:]) {
 		t.Fatalf("channel state not properly saved")
 	}
-
-	// Assert that the channel opens after a single block.
-	lnChan := make(chan *lnwallet.LightningChannel, 1)
-	go func() {
-		openDetails, err := chanReservation.DispatchChan()
-		if err != nil {
-			t.Fatalf("unable to finalize reservation: %v", err)
-		}
-
-		lnChan <- openDetails.Channel
-	}()
-	lnc := assertChannelOpen(t, miner, uint32(numReqConfs), lnChan)
-
-	// Now that the channel is open, execute a cooperative closure of the
```

Comment:
I can see possibly moving this section to a new unittest within `channel_test.go`. But until then it can be retained here.